pkg/backend/apply.go

```diff
@@ -125,6 +125,17 @@ func PreviewThenPrompt(ctx context.Context, kind apitype.UpdateKind, stack Stack
 		return plan, changes, nil
 	}
 
+	infoPrefix := "\b" + op.Opts.Display.Color.Colorize(colors.SpecWarning+"info: "+colors.Reset)
+	if kind != apitype.UpdateUpdate {
+		// If not an update, we can skip displaying warnings
+	} else if countResources(events) == 0 {
+		// This is an update and there are no resources being CREATED
+		fmt.Print(infoPrefix, "There are no resources in your stack(other than the stack resource).\n\n")
```

Comment:
@susanev I've incorporated your feedback into this version of the error message.
- add newline
- mention stack resource

@mikhailshilkov does this read better?